invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/ItemsSearch.js

```diff
@@ -43,6 +43,24 @@ export default class ItemsSearch extends Component {
     }
   };
 
+  _onPasteHandler = e => {
+    let queryString = e.clipboardData.getData('Text');
+    this.executeSearch(queryString).then(data => {
+      this.checkoutItem(this.props.items.hits[0], this.props.patron).then(
```

Comment:
If the pasted barcode results in no matches this will raise an error. Is that something we should guard against?

Reply:
yes, the bug was introduces with changes of error handling, I will create a ticket for it